microprofile/messaging/src/main/java/io/helidon/microprofile/messaging/CompletableQueueOverflowException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c)  2020 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.helidon.microprofile.messaging;
+
+/**
+ * Raised when {@link io.helidon.microprofile.messaging.CompletableQueue} max size is reached.
+ */
+public class CompletableQueueOverflowException extends RuntimeException {
```

Comment:
Do we really need so many types of exceptions? 
I think we do not expect anybody to catch these exceptions specifically, so we should be fine with a `MessagingException` and if needed a `MessagingDeploymentException` and describe the problem in `message`, rather then creating a typed exception for each case.

Reply:
fixed